databricks/koalas/base.py

```diff
@@ -184,8 +184,14 @@ def __sub__(self, other):
             return _column_op(spark.Column.__sub__)(self, other)
 
     __mul__ = _column_op(spark.Column.__mul__)
-    __div__ = _numpy_column_op(spark.Column.__div__)
-    __truediv__ = _numpy_column_op(spark.Column.__truediv__)
+
+    def __truediv__(self, other):
+        def truediv(left, right):
```

Comment:
`left` and `right` must be both Spark Column or a scalar value.

Reply:
I mean, we don't need to wrap with `F.lit()` for most cases. how about:

```diff
     def __truediv__(self, other):
         def truediv(left, right):
-            return F.when(F.lit(right) == 0, F.lit(np.inf).__div__(F.lit(left))).otherwise(
-                F.lit(left).__truediv__(F.lit(right))
+            return F.when(F.lit(right == 0), F.lit(np.inf).__div__(left)).otherwise(
+                left.__truediv__(right)
             )

         return _numpy_column_op(truediv)(self, other)
```

Reply:
@ueshin Oh I got it! Thanks for the example :D